pkgs/racket-doc/scribblings/reference/dicts.scrbl

```diff
@@ -1014,4 +1015,51 @@ See also @racket[define-custom-hash-types].
 
 }
 
+@section{Passing keyword arguments in dictionaries}
+
+@defproc[
+ (keyword-apply/dict [proc procedure?]
+                     [kw-dict (dict/c keyword? any/c)]
```

Comment:
What is `dict/c`? I searched and only found one from `mischief` which I don't think is what you want.

Reply:
Oh, I had assumed `(dict/c keyword? any/c)` was like `(hash/c keyword? any/c)`, but for generic dicts and not just hash tables. But I suppose if it doesn't exist, maybe just `dict?` should go there? Although I would prefer it to say the keys must be keywords... would `(and/c dict? (compose (listof keyword?) dict-keys))` work better? Kinda long though.

Reply:
Okay, I've changed it to `dict?` for now